carma-messenger-core/cpp_message/src/MobilityResponse_Message.cpp

```diff
@@ -0,0 +1,231 @@
+/*
```

Comment:
I thought the response stuff would be in a separate PR, is this intended to be here?

Reply:
This one is the MobilityResponse PR. It builds on top of the MobilityOperation branch which is why it includes both.